packages/desktop/src/renderer/components/CreateJoinCommunity/PerformCommunityActionComponent.tsx

```diff
@@ -224,6 +224,8 @@ export const PerformCommunityActionComponent: React.FC<PerformCommunityActionPro
   // TODO: What does this mean and do we need this here? It might make
   // sense to decouple the PSK from this component, since this is the
   // only place PSK is used.
+
+  // TODO: adjust to v2 link, maybe pass already composed link instead of data
```

Comment:
I'll just remove code responsible for locking form. It hasn't been used since orbitdbidentity was added. One added ownerOrbitDbIdentity to PerformCommunityActionComponent props but did not pass down value from JoinCommunity.